src/main/java/com/extendedclip/deluxemenus/utils/Constants.java

```diff
@@ -42,6 +42,8 @@ private Constants() {
     public static final String ITEMSADDER_PREFIX = "itemsadder-";
     public static final String ORAXEN_PREFIX = "oraxen-";
     public static final String MMOITEMS_PREFIX = "mmoitems-";
+    public static final String EXECUTABLEITEMS_PREFIX = "execitems-";
+    public static final String EXECUTABLEBLOCKS_PREFIX = "execblocks-";

```

Comment:
Are these constants used anywhere anymore? If yes, they should be changed. If no, they should be removed.